modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -34,23 +34,24 @@ If the cluster is configured with an intermediate certificate, then the whole ch
 
 == Adding new CA certificates into {prod-short}
 
-The following procedure is applicable for already installed and running instances and for instances that are to be installed.
+Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`and others. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
 
 .Prerequisites
 
 * An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 * Namespace for {prod-short} exists.
+* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
+  ** `ca-bundle.crt`
+  ** `ca.crt`
 
 .Procedure
 
 . Save the certificates you need to import, to a local file system.
 +
 [CAUTION]
 ====
-* Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But, they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
-* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
-  ** `ca-bundle.crt`
-  ** `ca.crt`
+* A certificate with the `BEGIN TRUSTED CERTIFICATE` introductory phrase is likely in the PEM `TRUSTED CERTIFICATE` format, which is not supported. Convert it to the supported `CERTIFICATE` format with the following command:
```

Comment:
I am confused. Does it contradictory to:

```
Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`and others. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
```

Reply:
As I understand the line, it explains what formats certificates have and which format should be used.

Reply:
I mean, here we mention, that `.cer` extension is used for binary-encoded certificate format which are not supported:

```
 Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
```

and below, we suggest to convert certificate using file exertion that is not supported accordingly to phrase above.

```
openssl x509 -in cert.pem -out cert.cer
```

It sounds inconsistent for me.

Reply:
Yes, that's will be better from my point of view.

Reply:
Done